specs/validator/beacon_node_oapi.yaml

```diff
@@ -0,0 +1,609 @@
+                pattern: "^0x[a-fA-F0-9]{96}$"
+                description: "Sender withdrawal public key."
+              signature:
+                type: string
+                format: byte
+                pattern: "^0x[a-fA-F0-9]{192}$"
+                description: "Sender signature."
+
+    Fork:
+      type: object
+      description: "The [`Fork`](https://github.com/ethereum/eth2.0-specs/blob/dev/specs/core/0_beacon-chain.md#Fork) object from the Eth2.0 spec."
+      properties:
+        previous_version:
+          type: string
+          format: byte
+          pattern: "^0x[a-fA-F0-9]{8}$"
+          description: "Previous fork version."
+        current_version:
+          type: string
+          format: byte
+          pattern: "^0x[a-fA-F0-9]{8}$"
+          description: "Current fork version."
+        epoch:
+          type: integer
+          format: uint64
+          description: "Fork epoch number."
+    IndexedAttestation:
+      type: object
+      description: "The [`IndexedAttestation`](https://github.com/ethereum/eth2.0-specs/blob/dev/specs/core/0_beacon-chain.md#indexedattestation) object from the Eth2.0 spec."
+      properties:
+        custody_bit_0_indices:
+          type: array
+          description: "Validator indices for 0 bits."
+          items:
+            type: integer
+            format: uint64
+        custody_bit_1_indices:
+          type: array
+          description: "Validator indices for 1 bits."
+          items:
+            type: integer
+            format: uint64
+        data:
+          $ref: '#/components/schemas/AttestationData'
+    AttestationData:
+      type: object
+      description: "The [`AttestationData`](https://github.com/ethereum/eth2.0-specs/blob/dev/specs/core/0_beacon-chain.md#attestationdata) object from the Eth2.0 spec."
+      properties:
+        beacon_block_root:
+          type: string
+          format: byte
+          pattern: "^0x[a-fA-F0-9]{64}$"
+          description: "LMD GHOST vote."
+        source_epoch:
+          type: integer
+          format: uint64
+          description: "Source epoch from FFG vote."
+        source_root:
+          type: string
+          format: byte
+          pattern: "^0x[a-fA-F0-9]{64}$"
+          description: "Source root from FFG vote."
+        target_epoch:
+          type: integer
+          format: uint64
+          description: "Target epoch from FFG vote."
+        target_root:
+          type: string
+          format: byte
+          pattern: "^0x[a-fA-F0-9]{64}$"
+          description: "Target root from FFG vote."
+        crosslink:
+          title: CrossLink
+          type: object
+          description: "The [`Crosslink`](https://github.com/ethereum/eth2.0-specs/blob/dev/specs/core/0_beacon-chain.md#crosslink) object from the Eth2.0 spec."
+          properties:
+            shard:
+              type: integer
+              format: uint64
+              description: "The shard number."
+            epoch:
```

Comment:
`epoch` has been removed and `start_epoch` and `end_epoch` have been added to `Crosslink` in `dev` branch